app/connectors/EventReportingConnector.scala

```diff
@@ -107,7 +109,7 @@ class EventReportingConnector @Inject()(
   }
 
   def submitReport(pstr: String, ua: UserAnswers, version: String)
-                  (implicit headerCarrier: HeaderCarrier): Future[Unit] = {
+                  (implicit headerCarrier: HeaderCarrier): Future[Result] = {
 
```

Comment:
Change submitReport return type to Future[HttpResponse] and handle via controller.